therapist/cli.py

```diff
@@ -212,32 +216,48 @@ def run(**kwargs):
         runner = Runner(repo_root, **kwargs)
     except Runner.Misconfigured as err:
         output(MISCONFIGURED_MSG.format(err.message))
+
+        if err.code == Runner.Misconfigured.PLUGIN_NOT_INSTALLED:
+            output('Installed plugins:')
+            for p in list_plugins():
+                output(p)
+
         exit(1)
     else:
         results = ResultCollection()
 
         if runner.unstaged_changes and not quiet:
             output(UNSTAGED_CHANGES_MSG, end='\n\n')
 
-        actions = [a.name for a in runner.actions]
+        processes = [a for a in runner.actions]
+        processes += [p for p in runner.plugins]
```

Comment:
Why the list comprehensions here? Could you just do `processes = runner.actions + runner.plugins`? If they are generators, you could `list(runner.actions)` instead.